CHANGELOG.md

```diff
@@ -23,14 +23,20 @@ changes in the following format: PR #1234***
   and transfer object (PR #7817)
 - BVL Feedback widget only shows notifications for the users sites / projects (PR #7848)
 - Add Date status change value in session table (PR #8350)
+- Fixed the Candidate Age at Death field label and Data Dictionary item for LINST instruments (PR #8362)
 
 #### Bug Fixes
 - placeholder
 
 ### Modules
-#### ModuleName
-- placeholder
 
+#### API
+- Modified in v0.0.4-dev the candidate instrument data format returned by a GET request or
```

Comment:
this item deserrves to be in notes for existing projects as well, just in case someone is using it